code/game/machinery/computer/arcade.dm

```diff
@@ -1159,6 +1170,13 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	else
 		to_chat(c_user, "<span class='notice'>You (wisely) decide against putting your hand in the machine.</span>")
 
+/obj/machinery/computer/arcade/proc/mousevend(mob/user)
```

Comment:
If I'm honest it may just be better for you to add this in pricevend() itself, since this will display 3x messages,

Reply:
3 messages good